spec/shr_core_vs.txt

```diff
@@ -351,37 +351,37 @@ Includes codes descending from SCT#123037004  "Body Structure"
 ValueSet:       AnatomicalDirectionVS
 Concept:        NCI#C99074
 Description:    "Anatomical location or specimen further detailing directionality. Codes are from NCI"
-#C25231 "ANTERIOR Denoting the front portion of the body or a structure."
-#C25423 "APICAL Relating to or located at the apex."
-#C90067 "BASAL Relating to or located at the lowest portion of a structure."
-#C73851 "CAUDAL Toward the tail in a body."
-#C25445 "CENTRAL A point or area that is approximately central within some larger region or structure."
-#C37936 "CRANIAL Toward the head in a body."
-#C25240 "DEEP Extending relatively far inward."
-#C25237 "DISTAL Situated farthest from a point of reference."
-#C45874 "DORSAL Pertaining to the back or upper surface of the body."
-#C90376 "DORSOLATERAL Toward the back and side of a body."
-#C90386 "FORE Of or involving the front of a main body."
-#C90393 "HIND Of or involving the back of a main body."
-#C25353 "INFERIOR Pertaining to a point below a given reference point."
-#C37980 "INNER Inside or closer to the inside of the body or object."
-#C73705 "INTERMEDIATE Located between two points or extremes."
-#C25309 "LOWER The bottom one of two."
-#C25232 "MEDIAL Toward the middle or in a limb toward the median plane."
-#C81170 "MIDLINE A medial line, especially the medial line or medial plane of the body (or some part of the body)."
-#C27958 "NASAL Of, or related to, or in the direction of the nose."
-#C38166 "OUTER Being on or toward the outside of the body or object."
-#C25233 "PERIPHERAL On or near an edge or constituting an outer boundary;the outer area."
-#C25622 "POSTERIOR Denoting the back portion of the body or a structure."
-#C94393 "ROSTRAL Toward the muzzle in the head."
-#C25239 "SUPERFICIAL Of or pertaining to the exterior surface."
-#C25235 "SUPERIOR Pertaining to a point above a given reference point."
-#C25245 "SURFACE The extended two-dimensional outer layer or area of a three-dimensional object."
-#C117754 "TEMPORAL Of, or related to, anatomic sites that are located in the cranium, brain, and eye."
-#C90069 "TIP The pointed end of a structure."
-#C25355 "UPPER The top one of two."
-#C45875 "VENTRAL Pertaining to the front or lower surface of the body."
-#C98798 "VENTROLATERAL Of or pertaining to the front and side of a main body."
+NCI#C25231 "ANTERIOR Denoting the front portion of the body or a structure."
```

Comment:
In some cases we use NCIt (shown as NCI#) and others as the NCI Metathesaurus (shown as MTH#).  In prior discussions with Rute, I think the preference is for the NCI metathesaurus.
Not urgent to fix now, but we might want to choose one convention for consistency.